src/common/low_precision_transformations/include/low_precision/network_helper.hpp

```diff
@@ -318,19 +318,12 @@ template <typename T, typename... Args>
 std::shared_ptr<Node> fold_reshape(Args&&... args) {
```

Comment:
If I am not mistaken, the idea of fold_reshape method was in avoiding reshape folding in case it's not needed. Namely, in most cases both inputs are constants and we can use Constant constructor instead of more cumbersome reshape folding